### PR TITLE
chore(dbt-fal): add dbt-fal publishing

### PR DESCRIPTION
**Pull request by chamini2**
## Description
This is designed to publish each package at a time.

One problem is changelog generation will be weird. We probably will need to start separating commits like:

- `fix(fal): ...`: fal-cli only
- `fix(dbt-fal): ...` or `fix(adapter): ...`: adapter only
- `fix: ...` all

### CODE REVIEW

**Comment by chamini2** [comment]
image

**Review by turbo1912 — APPROVED** [review]

File: .github/workflows/python-release.yml [hunk 1]
```diff
@@ -5,6 +5,14 @@ name: PyPI Release
...
   workflow_dispatch:
     inputs:
+      package:
+        description: Package
+        required: false
+        default: fal
+        type: choice
+        options:
+          - fal
+          - dbt-fal
```
Comment (isidentical) [inline .github/workflows/python-release.yml:15]: This is super cool (being able to choose which to publish)

**Review by isidentical — APPROVED** [review]
Agree on the changelog point.

File: .github/workflows/python-release.yml [hunk 2]
```diff
@@ -35,11 +48,23 @@ jobs:
...
 
+      - name: Package setup
+        run: |
+          if [[ ${{ inputs.package }} == fal ]]
+          then
+            "PACKAGE_DIR=." >> $GITHUB_ENV
+            "TAG_PREFIX=v" >> $GITHUB_ENV
+          else if [[ ${{ inputs.package }} == dbt-fal ]]
+            "PACKAGE_DIR=adapter" >> $GITHUB_ENV
+            "TAG_PREFIX=adapter-v" >> $GITHUB_ENV
```
Comment (isidentical) [inline .github/workflows/python-release.yml:59]: This is super clever btw 💯

**Comment by isidentical** [comment]
@chamini2 git cliff seems to have a path filter option

```
    --include-path <PATTERN>...    Sets the path to include related commits [env: GIT_CLIFF_INCLUDE_PATH=]
    --exclude-path <PATTERN>...    Sets the path to exclude related commits [env: GIT_CLIFF_EXCLUDE_PATH=]
```

I wonder, at least at the beginning, whether it is easier to configure in terms of paths instead of commit messages?

**Pull request merged**